charts/rancher-turtles/requirements.yaml

```diff
@@ -0,0 +1,5 @@
+dependencies:
+  - name: cluster-api-operator
+    version: v0.5.1
```

Comment:
Could this version be parametrized?

Reply:
It depends, we can pass it from Makefile by patching this file or if we want to have a somehow functional chart in `charts`(not like in capi operator where it's assembled on release command) just leave it as it is. The same applies to keeping CAPI operator chart tarball.